src/promptflow/tests/sdk_cli_test/e2etests/test_cli.py

```diff
@@ -1702,3 +1702,22 @@ def test_unknown_command(self, capfd):
             pass
         _, err = capfd.readouterr()
         assert "invalid choice" in err
+
+    def test_config_set_user_agent(self) -> None:
+        run_pf_command(
+            "config",
+            "set",
+            "user_agent=test/1.0.0",
```

Comment:
user_agent = "test/1.0.0" on the PR title.
and user_agent = test/1.0.0 in this test case

Reply:
Offline synced, we'll keep what user specified and won't do escape for it: 
![image](https://github.com/microsoft/promptflow/assets/7776147/40e36268-58f6-4728-b08c-bfae2d2da089)